packages/server/package.json

```diff
@@ -4,33 +4,45 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
+    "postinstall": "link-module-alias",
```

Comment:
It's not necessary with yarn workspace and lerna. The module alias is already available by creating new directory in packages.

Reply:
You can look on this example https://github.com/Romakita/tsed-lerna-example/
It use package and Ts.ED module to load Services and Controller :)

Reply:
Another advantage, IDE discover correctly the modules because, the all packages are linked into node_modules. This is not the case with link-module-alias (it's a hack of require). :)

Reply:
Done.